Dockerfile

```diff
@@ -18,17 +18,14 @@ COPY --from=BUILDER /workdir/_output/bin/mcad-controller /usr/local/bin
 RUN true \
     && microdnf update \
     && microdnf --nodocs install \
-        curl shadow-utils \
+        curl \
     && microdnf clean all \
     && true
 
-RUN cd /usr/local/bin && curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl
+RUN cd /usr/local/bin && curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/latest.txt)/bin/linux/amd64/kubectl && chmod +x kubectl
```

Comment:
@jbusche can we move the download of the kubectl and installation into the builder stage ? I think that `curl` might be available in the builder.

Reply:
Hi @z103cb, yes, that seemed to work nicely...  by doing that I was able to eliminate the installation of curl completely.